### ISSUE TITLE
Toggling the price filter toggles the update button as well

### ISSUE THREAD

**Issue by NyanKiyoshi** [mirumee/saleor#2168]
Hi,

There is an issue with the price filter: toggling it hides the update button as well.

![video](https://i.imgur.com/aTBtfao.gif)

**Comment by patrys** [comment]
I think the update button is only for the price filter? Other filters should apply immediately, at least on the demo branch where React is used.

**Comment by NyanKiyoshi** [comment]
On the demo branch, yes. But on master we have to click on the button, same thing happens when we have other filters.

![scr](https://i.imgur.com/Thkprk2.png)

Kinda relate to what was said on #2118.

**Comment by NyanKiyoshi** [comment]
There is also an issue with the submit button (`.btn.primary`) it has `padding: 1rem 4rem` which is quite ugly in French, we probably should replace that by `padding: 1rem`

`1rem 4rem` padding:
![scr](https://i.imgur.com/FrKa4KE.png)

`1rem` padding:
![scr](https://i.imgur.com/R1nkMx4.png)

---

And, to fix #2168, it would require to move down the button:
```diff
Index: templates/product/_filters.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- templates/product/_filters.html	(revision 8857c3c7b21e641382c6a8e77dc6bac171d75625)
+++ templates/product/_filters.html	(date 1526378744000)
@@ -60,9 +60,10 @@
                         value="{% if field.value.1 %}{{ field.value.1 }}{% endif %}" type="number" min="0"
                         class="form-control d-inline" placeholder="{% trans 'to' context 'Product price filter' %}"/>
                 </div>
-                <button class="btn primary" type="submit">{% trans 'Update' context 'Product price filter' %}</button>
               </div>
             </div>
+
+            <button class="btn primary" type="submit">{% trans 'Update' context 'Product price filter' %}</button>
           {% endif %}
         {% endfor %}
       </form>
```
